src/ontology/edits/oeo-shared-axioms.omn

```diff
@@ -3997,135 +3996,33 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1463"@en,
-    
-    EquivalentTo: 
-        OEO_00000147
-         and (<http://purl.obolibrary.org/obo/RO_0000057> some OEO_00000006)
-    
-    SubClassOf: 
-        OEO_00000147
-    
     
 Class: OEO_00320000
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A transport performance value is a quantity value that indicates the performance of a transport process in terms of its mileage and amount of transported people and/or goods."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289"@en,
-        rdfs:label "transport performance value"@en
-    
-    SubClassOf: 
-        OEO_00000350,
-        OEO_00040010 some OEO_00320001
-    
     
 Class: OEO_00320001
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A transport performance unit is a unit of measurement for the accumulated transport distance of a number of people and/or amount of goods.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289",
-        rdfs:label "transport performance unit"
-    
-    SubClassOf: 
-        <http://purl.obolibrary.org/obo/UO_0000000>
-    
     
 Class: OEO_00320002
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Passenger-kilometre is a transport performance unit for the accumulated transport distance of people where one passenger-kilometre equals the transport distance of 1 km for one person.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289
-
-Add pkm as synonym:
-issue: https://github.com/OpenEnergyPlatform/ontology/issues/1375
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1388",
-        <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "pkm",
-        rdfs:label "passenger-kilometre"
-    
-    SubClassOf: 
-        OEO_00320001
-    
     
 Class: OEO_00320003
 
-    Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Ton-kilometre is a transport performance unit for the accumulated transport distance of goods where one ton-kilometre equals the transport distance of 1 km for one ton of goods.",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1272
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1289
-
-Add tkm as synonym:
-issue: https://github.com/OpenEnergyPlatform/ontology/issues/1375
-pull-request: https://github.com/OpenEnergyPlatform/ontology/pull/1388",
-        <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> "tkm",
-        rdfs:label "ton-kilometre"
-    
     SubClassOf: 
         OEO_00320001
```

Comment:
This axiom (`ton-kilometre SubClassOf: 'transport performance unit'`) appears twice and should be removed from oeo-shared-axioms